includes/ucf-social-common.php

```diff
@@ -0,0 +1,145 @@
+<?php
+/**
+ * Place common functions here.
+ **/
+
+if ( ! class_exists( 'UCF_Social_Common' ) ) {
+	class UCF_Social_Common {
+
+		/**
+		* Displays the social social icons
+		* @author RJ Bruneel
+		* @since 1.0
+		* @param array $atts | Assoc. array of shortcode options
+		* @return string
+		**/
+		public static function display_social_icons( $atts ) {
+				$before = self::ucf_social_icons_display_before();
+				if ( has_filter( 'ucf_social_display_before' ) ) {
+					$before = apply_filters( 'ucf_social_display_before', $before );
+				}
+
+				$content = self::ucf_social_icons_display( $atts );
+				if ( has_filter( 'ucf_social_display' ) ) {
+					$content = apply_filters( 'ucf_social_display', $content );
+				}
+
+				$after = self::ucf_social_icons_display_after( );
+				if ( has_filter( 'ucf_social_display_after' ) ) {
+					$after = apply_filters( 'ucf_social_display_after', $after );
+				}
+
+				return $before . $content . $after;
+
+		}
+
+		/**
+		* Display the content before the social icons
+		* @author RJ Bruneel
+		* @since 1.0
+		* @return string
+		**/
+		function ucf_social_icons_display_before() {
+			ob_start();
+		?>
+			<div class="ucf-social-icons">
+		<?php
+			echo ob_get_clean();
+		}
+
+		/**
+		* Display the social icons content
+		* @author RJ Bruneel
+		* @since 1.0
+		* @param array $atts | Assoc. array of shortcode options
+		* @return string
+		**/
+		function ucf_social_icons_display( $atts ) {
+				$atts = shortcode_atts( array(
+					'size'  => 'md',
+					'color' => 'color'
+				), $atts );
+
+			$prefix = UCF_Social_Config::$option_prefix;
+
+			$google_url     = UCF_Social_Config::get_option_or_default( $prefix . 'google_url' );
```

Comment:
I might be wrong about this, but I think `get_option_or_default` will work with or without the prefix. It does in some of the other plugins we've built.